bundles/org.openhab.binding.worxlandroid/src/main/resources/OH-INF/i18n/worxlandroid.properties

```
@@ -0,0 +1,355 @@
+thing-type.worxlandroid.mower.description = Represents a Landroid Worx Mower
+thing-type.worxlandroid.mower.group.friday.label = Friday Schedule
+thing-type.worxlandroid.mower.group.friday2.label = Friday Slot 2 Schedule
+thing-type.worxlandroid.mower.group.monday.label = Monday Schedule
+thing-type.worxlandroid.mower.group.monday2.label = Monday Slot 2 Schedule
+thing-type.worxlandroid.mower.group.saturday.label = Saturday Schedule
+thing-type.worxlandroid.mower.group.saturday2.label = Saturday Slot 2 Schedule
+thing-type.worxlandroid.mower.group.sunday.label = Sunday Schedule
+thing-type.worxlandroid.mower.group.sunday2.label = Sunday Slot 2 Schedule
+thing-type.worxlandroid.mower.group.thursday.label = Thursday Schedule
+thing-type.worxlandroid.mower.group.thursday2.label = Thursday Slot 2 Schedule
+thing-type.worxlandroid.mower.group.tuesday.label = Tuesday Schedule
+thing-type.worxlandroid.mower.group.tuesday2.label = Tuesday Slot 2 Schedule
+thing-type.worxlandroid.mower.group.wednesday.label = Wednesday Schedule
+thing-type.worxlandroid.mower.group.wednesday2.label = Wednesday Slot 2 Schedule
+
+# thing types config
+
+thing-type.config.worxlandroid.bridge.password.label = Password
+thing-type.config.worxlandroid.bridge.password.description = Password to access the Landroid WebAPI.
+thing-type.config.worxlandroid.bridge.username.label = Username
+thing-type.config.worxlandroid.bridge.username.description = Username to access the Landroid WebAPI.
+thing-type.config.worxlandroid.mower.pollingInterval.label = Polling Interval
+thing-type.config.worxlandroid.mower.pollingInterval.description = Interval for polling in seconds
+thing-type.config.worxlandroid.mower.refreshStatusInterval.label = Refresh Status Interval
+thing-type.config.worxlandroid.mower.refreshStatusInterval.description = Interval for refreshing mower status in seconds
+thing-type.config.worxlandroid.mower.serialNumber.label = Serial Number
+thing-type.config.worxlandroid.mower.serialNumber.description = Serial number of the mower
+
+# channel group types
+
+channel-group-type.worxlandroid.aws-group-type.label = Aws
+channel-group-type.worxlandroid.aws-group-type.description = MQTT connexion to AWS
+channel-group-type.worxlandroid.aws-group-type.channel.connected.label = Connected
+channel-group-type.worxlandroid.aws-group-type.channel.connected.description = Connection to AWS is alive
+channel-group-type.worxlandroid.aws-group-type.channel.poll.label = Poll AWS
+channel-group-type.worxlandroid.aws-group-type.channel.poll.description = Enables or disables polling Worx AWS
+channel-group-type.worxlandroid.battery-group-type.label = Battery
+channel-group-type.worxlandroid.battery-group-type.description = Battery channels of your mower
+channel-group-type.worxlandroid.battery-group-type.channel.charge-cycles.label = Current Charge Cycles
+channel-group-type.worxlandroid.battery-group-type.channel.charge-cycles-total.label = Total Charge Cycles
```

=> mvn i18n:generate-default-translations

```suggestion
channel-group-type.worxlandroid.battery-group-type.channel.charge-cycles.label = Charge Cycles
```

---

Corrected